caikit/runtime/service_generation/primitives.py

```diff
@@ -82,6 +82,36 @@ def to_primitive_signature(
     return primitives
 
 
+def to_output_dm_type(arg_type: type) -> str:
```

Comment:
Should this be `to_output_dm_type(arg_type: Type) -> Type:`?
Looks like it takes a type and returns a type

Reply:
Could even throw one of those "complete but long and awful" names on here like `extract_data_model_type_from_union` 🤔